simapp/app.go

```diff
@@ -280,10 +278,7 @@ func NewSimApp(
 	evidenceKeeper := evidencekeeper.NewKeeper(
```

Comment:
Should I just remove the `evidenceKeeper` entirely since it isn't really used anymore

Reply:
@clevinson ?

Reply:
can you create an issue? Let's discuss it there